packages/backend/src/payment-method/ilp/connector/core/middleware/ildcp.ts

```diff
@@ -22,7 +22,7 @@ export function createIldcpMiddleware(serverAddress: string): ILPMiddleware {
 
     const clientAddress = incoming.staticIlpAddress
     if (!clientAddress) {
-      logger.warn(
+      logger.error(
```

Comment:
For logging where there are errors, I generally made them error level like this. There were a few exceptions like the rate limiting errors. Those are currently warnings and I left them alone - warning felt appropriate there.